R/print.r

```diff
@@ -456,8 +456,9 @@ print.control.net <- function(x, ...) {
   if(is.null(x$module.order)) {
     cat("Dynamic Modules:", funToPrint)
   } else {
-    cat("Dynamic Modules:", intersect(x$module.order,
-                                      funToPrint))
+    order <- unlist(lapply(funToPrint, function(y) which(y == x$module.order)))
+    funToPrint <- funToPrint[order]
+    cat("Dynamic Modules:", funToPrint)
```

Comment:
@smjenness can I ask what the difference between this and printing module.order should be (identical at this point). Should it just print the module order, or the module order then concatenated with remaining modules that run? Hopefully this makes sense.

Reply:
the print function should print out all the modules (except for initialize and verbose) in the order in which they are run within the netsim time loop.

Reply:
@smjenness updated the function but I'm not sure if this situation would ever exist: print takes module order and then concatenates with the default order of the other modules to get the full module run order; this is in case someone doesn't pass full list or modules into module.order parameter. If this is over-gimmicked can revert to just looking at module.order.

Reply:
Thanks. If ready to review pls click that button on the PR